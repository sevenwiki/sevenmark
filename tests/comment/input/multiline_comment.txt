Text before comment /* This is a
multiline comment
that spans multiple lines */ text after comment.