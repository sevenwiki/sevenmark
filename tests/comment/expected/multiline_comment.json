[
  {
    "Text": {
      "location": {
        "start": 0,
        "end": 20
      },
      "content": "Text before comment "
    }
  },
  {
    "Comment": {
      "location": {
        "start": 20,
        "end": 79
      },
      "content": " This is a\nmultiline comment\nthat spans multiple lines "
    }
  },
  {
    "Text": {
      "location": {
        "start": 79,
        "end": 99
      },
      "content": " text after comment."
    }
  }
]